spec.md

```diff
@@ -151,6 +151,21 @@ Service Broker MAY reject the request with `412 Precondition Failed` and
 provide a message that informs the operator of the API version that is to be
 used instead.
 
+## Extending Resources
+
+Senders of messages defined by this specification MAY include additional
+fields within the JSON objects as extensions. For backwards compatibility
+reasons, new fields MAY be added at any location within the JSON objects,
+however, they SHOULD be placed within the `extensions` property of the JSON
+object.
```

Comment:
The wording I would like to see is more the the following:

> Senders of messages defined by this specification MAY include additional
fields within the JSON objects. Only fields found within the `extensions` location
are considered extensions. For backwards compatibility reasons, new fields MAY
be added at any location within the JSON objects, but these MAY be ignored by
the Platform. Extensions MUST be placed within the `extensions` property of the JSON
object.

Basically I don't want to limit the data returned from the broker, but I can't see a future compatible way to make this work if we don't direct that output.